docs/_toc.yml

```diff
@@ -0,0 +1,10 @@
+# Table of contents
+# Learn more at https://jupyterbook.org/customize/toc.html
+
+format: jb-book
+root: intro
+parts:
+- caption: Getting Started
+  chapters:
+    - title: Installation
+      file: installation
```

Comment:
What sections/subsections should we put on the sidebar? My initial ideas are:

- Getting started
  - Installation
- Data Preparation
  - Preparing the datacube
  - Preparing labels for evaluation model performance
- Running the model
  - Self-supervised learning
    - Linear Probing?
  - Generating embeddings

Reply:
- I would add a paragraph about the clay foundation and a link to the [linkedin page](https://www.linkedin.com/company/made-with-clay).
- We could add a section about how to run the datacube in batch

Reply:
Ok, added a link to the LinkedIn (and GitHub) page at commit bcbf080028c7067d658e3a4a758e7e9149a4e84d. Gonna merge this PR soon, so I'll open a separate issue to discuss about what sections/sub-sections to add later.